Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -197,8 +414,7 @@ public struct HTTPHeaders: CustomStringConvertible {
     //      recommended.
     /// - Parameter value: The header field value to add for the given name.
     public mutating func add(name: String, value: String) {
```

Comment:
@weissi @Lukasa also as a followup I think we should support `StaticString`

Reply:
What would that support look like?